src/beast/beast/module/core/native/linux_Files.cpp

```diff
@@ -201,8 +200,8 @@ class DirectoryIterator::NativeIterator::Pimpl
     DIR* dir;
 };
 
-DirectoryIterator::NativeIterator::NativeIterator (const File& directory, const String& wildCard)
-    : pimpl (new DirectoryIterator::NativeIterator::Pimpl (directory, wildCard))
+DirectoryIterator::NativeIterator::NativeIterator (const File& directory, const String& wildCard_)
```

Comment:
Apparently the trailing underscore is correct, but I've been ordered to abandon this so...

A shame, I thought some of these seemed like legitimate issues.